src/Codeception/Module/Symfony2.php

```diff
@@ -67,11 +70,11 @@
  * * container - dependency injection container instance
```

Comment:
One last change: As this property no longer exists as is, please remove it from documentation.